### PR TITLE
BAH Sprint 8: Updated the header, footer stamping text for the ancillary forms

### PR DESCRIPTION
**Pull request by abhijitvk**
## Description of change

**READY FOR THE MERGE**

Updated the text that is generated at the header and footer of the PDFs for the ancillary forms (21-4142, 21-4142A, 21-0781, 21-0781A so that it is helpful to the end user that will be receiving these PDFs after submission on the va.gov platform

Sample PDFs generated newly are attached to the connected story

## Testing done
- Unit testing
- Local testing

## Testing planned
- Unit testing
- Cross-browser (by QA)
- Manual testing (by QA)

## Acceptance Criteria (Definition of Done)
- Unit tests passed
    - spec/jobs/central_mail/submit_form4142_spec.rb
    - spec/jobs/evss/disability_compensation_form/submit_form0781_spec.rb

#### Unique to this PR
NA

#### Applies to all PRs

- [X] Appropriate logging
- [X] Swagger docs have been updated, if applicable
- [X] Provide link to originating GitHub issue, or connected to it via ZenHub
- [X] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [X] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by abhijitvk** [comment]
@kreek @egarreau @SMLuthi  -- Pretty straightforward minor changes for PDF stamping text.

File: app/workers/central_mail/submit_form4142_job.rb [hunk 1]
```diff
@@ -78,15 +78,15 @@ def to_faraday_upload(file_path)
 
     # Invokes Filler ancillary form method to generate PDF document
     # Then calls method CentralMail::DatestampPdf to stamp the document.
-    # Its called twice, once to stamp with text "VETS.GOV" at the bottom of each page
+    # Its called twice, once to stamp with text "VA.gov YYYY-MM-DD" at the bottom of each page
```
Comment (egarreau) [inline app/workers/central_mail/submit_form4142_job.rb]: Was the `YYYY-MM-DD` timestamp indicated in this comment always present, and the comment was just updated for clarity? Or has the timestamp been newly added? I don't see new code for that anywhere.

Comment (abhijitvk) [reply app/workers/central_mail/submit_form4142_job.rb]: Comment was just updated for clarity. Nothing new has been added.

**Comment by abhijitvk** [comment]
@kreek - let me know if this looks good to you and i can merge it

**Pull request merged**